docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -54,10 +54,12 @@ The top bar of Teleport Connect consists of:
   Teleport clusters as well as log in or out of the clusters.
 - The **connection list** (the top left) showing recent connections, allowing you to seamlessly switch
   between them.
-- The **command bar** (in the middle), which launches any command you input in a new terminal tab.
-- The **cluster selector** (to the left of the command bar), which shows up only if you have set up
+- The **search bar** (in the middle), which allows you to search for resources across clusters.
+- The **cluster selector** (to the left of the search bar), which shows up only if you have set up
   Trusted Clusters and there are leaf clusters connected to the root cluster. It lets you browse
-  leaf cluster resources. It also changes which cluster the commands in the command bar are going to target.
+  leaf cluster resources.
+- The **more options menu** (to the left of the profile selector), containing options such as
```

Comment:
idk, I feel like we cannot refer to it as "the more options menu" because it looks terrible in text. For example, when talking about the cluster selector I'd like to add this text:

> It also changes which cluster the local terminal tabs opened through "Open new terminal" from the more options menu are going to be bound to.

I searched through some old Apple docs, e.g. https://web.archive.org/web/20150818000225/https://support.apple.com/en-us/HT204839, and it looks like in their UI the More Options icon is always referred to in the context of another specific part of the UI.

FWIW, these are not options, each item refers to an action. But even if we call it "more actions" then "more actions menu" still looks bad.

What about renaming it to something like "additional actions menu"? Instead of saying "click the More Options icon" we'd say "open the additional actions menu" and then the user can refer to this section of the docs to understand what the "additional actions menu" is.

Reply:
I don't have a strong opinion on it, but I think this rename makes sense 👍